doc/ruleSets/PER-CS2.0.rst

```diff
@@ -0,0 +1,10 @@
+=======================
+Rule set ``@PER-CS2.0``
+=======================
+
+EXPERIMENTAL: Rules that follow `PER Coding Style 2.0 <https://www.php-fig.org/per/coding-style/>`_.
+
+Rules
+-----
+
+- `@PER-CS1.0 <./PER-CS1.0.rst>`_
```

Comment:
*question*

are all the rules and their config for PER-CS1.0 the same as for 2.0?
I.e. , if there is a rule that expects different style in v2 than it was in v1?

(same for sibling Risky set)

Reply:
If you look at #7247 then it's pretty obvious that works are divided between several atomic, independent tasks. This is the base for further works and yeah, we start from v1 🙂.

Reply:
my question was more about "is there a case like `v1 expects single space while v2 expects new line`, etc". 

gladly, If I see right, no style changed, v2 only adds style description for new language syntax
ref https://github.com/php-fig/per-coding-style/compare/1.0.0...master#diff-bc6661da34ecae62fbe724bb93fd69b91a7f81143f2683a81163231de7e3b545R4

Reply:
I don't *think* we had any BC-breaking style changes in v2.  We had some new "recommended"s that the v2 fixer should implement by default, but I don't believe there's any "just remove that rule entirely" changes.